docs/blog/2021-01-19-frontend-asset-passthrough.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Frontend Asset Passthrough
+authors:
+  - name: Mike Cutalo
+    url: https://github.com/mcutalo88
+    avatar: https://avatars1.githubusercontent.com/u/2250844?s=460&u=24deb32096e9f892cc91a6ff1ca1af50193b1fbd&v=4
+description: Frontend Asset Passthrough
+image: https://user-images.githubusercontent.com/4712430/104760766-7a2c5980-5727-11eb-93f5-3296b23ba3a0.png
```

Comment:
needs update

Reply:
Actually what is this for? the link preview?

Reply:
Ill use my envoy + clutch diagram

Reply:
yeah if it's posted to social, e.g. twitter i think it's the link preview